### PR TITLE
DCOS-39153: add tabs to NodesPage

### PR DESCRIPTION
**Pull request by juliangieseke**
Closes DCOS-39153

## Testing

navigate to http://localhost:4200/#/nodes/agents to see tabs. Agents page should work as before. masters tab is filled by #3117

### CODE REVIEW

**Comment by nLight** [comment]
Is this feature branch ready to be merged? It's in `in review` column.

File: plugins/nodes/src/js/pages/NodesAgents.js [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+  render() {
+    var data = this.internalStorage_get();
+    const statesProcessed = MesosSummaryStore.get("statesProcessed");
+    var isEmpty = statesProcessed && data.totalNodes === 0;
+
+    return this.getContents(isEmpty);
+  }
+});
+
+module.exports = NodesOverview;
```
Comment (DanielMSchmidt) [inline plugins/nodes/src/js/pages/NodesAgents.js]: I think this is now NodesAgent

**Comment by juliangieseke** [comment]
this isnt the feature branch… added the wrong label, sorry 😇

File: plugins/nodes/src/js/pages/NodesOverview.js [hunk 2]
```diff
@@ -204,22 +203,6 @@ var NodesOverview = React.createClass({
     );
   },
 
-  getFilterInputText() {
```
Comment (DanielMSchmidt) [inline plugins/nodes/src/js/pages/NodesOverview.js]: I don't understand what this change has to do with the rest? How does this contribute to having tabs for master and agent?

File: plugins/nodes/src/js/pages/NodesAgents.js [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+        <Badge>{count || 0}</Badge>
+      </span>
+    );
+  },
+
+  getViewTypeRadioButtons(resetFilter) {
+    const isGridActive = /\/nodes\/agents\/grid\/?/i.test(
+      this.props.location.pathname
+    );
+    // const activeView = this.props.location.pathname.includes("agents")
```
Comment (DanielMSchmidt) [inline plugins/nodes/src/js/pages/NodesAgents.js]: Please remove the comment

File: plugins/nodes/src/js/pages/NodesAgents.js [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+    //   ? "agents"
+    //   : "masters";
+
+    console.log(this.props.location);
```
Comment (DanielMSchmidt) [inline plugins/nodes/src/js/pages/NodesAgents.js]: Stray console.log

**Comment by nLight** [comment]
@juliangieseke then it looks like it has legit integration tests fails

**Comment by juliangieseke** [comment]
Looks like this branch wasnt as "done" as I expected it to be - sorry 😅 

@nLight @DanielMSchmidt comments addressed, test fixed, poor mans feature flag removed 👍

File: plugins/nodes/src/js/pages/NodesOverview.js (hunk 2 above)
Comment (DanielMSchmidt) [reply plugins/nodes/src/js/pages/NodesOverview.js]: Next time, please do a separate commit for that

**Pull request merged**